gpt_playground/urls.py

```diff
@@ -40,6 +40,7 @@
     path("pipelines/", include("apps.pipelines.urls")),
     path("files/", include("apps.files.urls")),
     path("annotations/", include("apps.annotations.urls", namespace="annotations")),
+    path("participants/", include("apps.participants.urls", namespace="participants")),
```

Comment:
`namespace` isn't needed here since you've included the `app_name` var in the `urls` module